browse-next/src/components/DeviceSetupDefineMask.vue

```diff
@@ -84,8 +91,8 @@ function formatRegionData(regionsArray) {
   for (let i = 0; i < regionsArray.value.length; i++) {
     const extractedPoints = regionsArray.value[i].regionData;
     const regionObj = {
-      "region": i.toString(),
-      "points": extractedPoints
+      region: i.toString(),
```

Comment:
Can we please implement user defined region names?  It may also make more sense for the stored regionData to be in the format { "name": pointsArray1, "name2": pointsArray2 }, so that we don't have to iterate through the array of regions to find the one we want to edit by name? 

I also still think that you may be over-complicating the code by having two separate storage formats one on the front-end and another on the back-end, and having to have code to convert between them.